docs/UserGuide.md

```diff
@@ -55,7 +55,7 @@ The guide uses the following features to make it easier for you to navigate arou
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Quick start
+## Quick Start
```

Comment:
Will it be better to standardise to sentence case instead. If not, I think need to change all the subheaders to title case.

Reply:
I took a look at some of the examples that Diana gave, and it seems that Title Case is used for all headings except those that are related to commands. As such for our user guide, I've standardised it to use Title Case for all heading levels 3 and above, while Sentence case is used for all heading levels 4 and below.